pymemcache/serde.py

```diff
@@ -55,11 +56,11 @@ def python_memcache_deserializer(key, value, flags):
         return int(value)
 
     if flags & FLAG_LONG:
-        return long(value)
+        return long_type(value)
```

Comment:
Not from what I can tell, it can only be set as `FLAG_LONG` line 39, and that only happens if `long_type` is explicitely not `None`